apps/opencs/model/world/refidadapterimp.cpp

```diff
@@ -431,7 +438,15 @@ void CSMWorld::MiscRefIdAdapter::setData (const RefIdColumn *column, RefIdData&
         InventoryRefIdAdapter<ESM::Miscellaneous>::setData (column, data, index, value);
 }
 
-CSMWorld::NpcColumns::NpcColumns (const ActorColumns& actorColumns) : ActorColumns (actorColumns) {}
+CSMWorld::NpcColumns::NpcColumns (const ActorColumns& actorColumns)
+: ActorColumns (actorColumns),
+  mFlags(std::map<const RefIdColumn *, unsigned int>()),
```

Comment:
This line might seem sort of pointless, but I've got the habit of initializing as much variables as possible in the initializer list. That's basically why this line is there, and it doesn't do any harm (except taking a few seconds to type it), since the map's default constructor would do something similar.

If that line bugs you, I sure can remove it (or you can do that during the merge, whatever is more convenient for you).